src/main/java/cloud/fogbow/ras/core/plugins/interoperability/aws/image/v2/AwsV2ImagePlugin.java

```diff
@@ -0,0 +1,87 @@
+package cloud.fogbow.ras.core.plugins.interoperability.aws.image.v2;
+
+import cloud.fogbow.common.exceptions.FogbowException;
+import cloud.fogbow.common.models.AwsV2User;
+import cloud.fogbow.common.util.PropertiesUtil;
+import cloud.fogbow.ras.api.http.response.Image;
+import cloud.fogbow.ras.core.models.ResourceType;
+import cloud.fogbow.ras.core.plugins.interoperability.ImagePlugin;
+import cloud.fogbow.ras.core.plugins.interoperability.aws.AwsV2ClientUtil;
+import cloud.fogbow.ras.core.plugins.interoperability.aws.AwsV2ConfigurationPropertyKeys;
+import cloud.fogbow.ras.core.plugins.interoperability.aws.AwsV2StateMapper;
+import software.amazon.awssdk.services.ec2.Ec2Client;
+import software.amazon.awssdk.services.ec2.model.*;
+
+import java.util.*;
+
+public class AwsV2ImagePlugin implements ImagePlugin<AwsV2User> {
+
+    private final Integer K_BYTES = 1024;
+    private final Integer K_BYTES_TO_GB = 3;
+    private final Integer NO_VALUE_FLAG = -1;
+    private final Integer FIRST_POSITION_VALUE = 0;
+
+    private Properties properties;
+    private String region;
+
+    public AwsV2ImagePlugin(String confFilePath) {
+        this.properties = PropertiesUtil.readProperties(confFilePath);
+        this.region = this.properties.getProperty(AwsV2ConfigurationPropertyKeys.AWS_REGION_SELECTION_KEY);
+    }
+
+    public Map<String, String> getAllImages(AwsV2User cloudUser) throws FogbowException {
+        DescribeImagesRequest imagesRequest = DescribeImagesRequest.builder().owners(cloudUser.getId()).build();
+        Ec2Client client = AwsV2ClientUtil.createEc2Client(cloudUser.getToken(), this.region);
+
+        DescribeImagesResponse imagesResponse = client.describeImages(imagesRequest);
+        Map<String, String> images = new HashMap<>();
+        List<software.amazon.awssdk.services.ec2.model.Image> retrievedImages = imagesResponse.images();
+
+        for(software.amazon.awssdk.services.ec2.model.Image image : retrievedImages) {
```

Comment:
Add spaces here to match the code standard we are using.

Reply:
Fixed!